src/commands/commit/get.ts

```diff
@@ -0,0 +1,82 @@
+import { Args, ux } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { formatDate } from "@/lib/helpers/date";
+import { withSpinner } from "@/lib/helpers/request";
+import { formatCommitAuthor } from "@/lib/marshal/commit";
+
+export default class CommitGet extends BaseCommand<typeof CommitGet> {
+  static summary = "Display a single commit based on an ID";
+
+  static args = {
+    id: Args.string({
+      required: true,
+    }),
+  };
+
+  static enableJsonFlag = true;
+
+  async run(): Promise<ApiV1.GetCommitResp | void> {
+    const { flags } = this.props;
+
+    const resp = await withSpinner<ApiV1.GetCommitResp>(() =>
+      this.apiV1.getCommit(this.props),
+    );
+
+    if (flags.json) return resp.data;
+    this.render(resp.data);
+  }
+
+  render(commit: ApiV1.GetCommitResp): void {
+    this.log(
+      `‣ Showing commit \`${commit.id}\` in \`${commit.environment}\` environment`,
+    );
+
+    /*
+     * Commit table
+     */
+
+    const rows = [
+      {
+        key: "ID",
+        value: commit.id,
+      },
+      {
+        key: "Resource",
+        value: commit.resource.type,
+      },
+      {
+        key: "Identifier",
+        value: commit.resource.identifier,
+      },
+      {
+        key: "Author",
+        value: formatCommitAuthor(commit),
+      },
+      {
+        key: "Commit message",
+        value: commit.commit_message ? commit.commit_message.trim() : "",
```

Comment:
I believe we still have the same issue here though, where a long commit message still gets truncated?

<img width="962" alt="CleanShot 2023-11-13 at 10 05 38@2x" src="https://github.com/knocklabs/knock-cli/assets/4471723/4321347c-eccd-404e-bde3-9397e9262493">

I'm thinking we might want to follow the similar format as git, where we basically print out the commit message separately at the bottom outside a table.

Reply:
Here's the commit: https://github.com/knocklabs/knock-cli/pull/263/commits/1dd6ca9ba4e3af0b1471bb29f2b4157d53be18ad

And this is how it looks now:

<img width="1440" alt="Screen Shot 2023-11-13 at 14 16 04" src="https://github.com/knocklabs/knock-cli/assets/67347884/7c48688e-1ae2-4e59-9b92-43521c323e24">
<img width="1438" alt="Screen Shot 2023-11-13 at 14 15 57" src="https://github.com/knocklabs/knock-cli/assets/67347884/c04276eb-dc86-4df0-9c65-b09c4af47554">

Another option could be to show `Commit Message: <commit.commit_message>`, but I think how we have it now looks better.